clasi_user/interface.py

```diff
@@ -0,0 +1,173 @@
+
+#################################################
+### THIS FILE WAS AUTOGENERATED! DO NOT EDIT! ###
+#################################################
+# file to edit: jupyter_notebooks/Test_module.ipynb
+
+import numpy as np
+import pandas as pd
+import os
+import json
+import random
+
+from fastai.text import *
+from fastai import *
+
+class Interface():
+    def __init__(self, path, bs=64, eval_mode=False):
+        '''
+        path must be a valid file path or file paths
+        path can be either str or Path obj
+        eval_mode set to False means create language model
+            and general classfier from scratch
+        '''
+        if not isinstance(path,(list,tuple)):
+            path = self._listify(path)     #if things are not iteratorable
+        self.path = self._clean_non_json_path(path) #remove non .json paths
+        self.csv_path = None
+        self.bs = bs
+        self.language_model_data = None
+        self.classification_model_data = None
+        self.eval_mode = eval_mode
+        self.df_train, self.df_valid = None, None
+        self.data_list, self.dataset_name = [], [] #to store all individual df for individual model
+
+    def _clean_non_json_path(self,path):
```

Comment:
I remember you said you have a script to generate the .py files from Jupiternotebook directly. It is still worth checking the coding style on the generated .py files. You can do this by running some style checker. Here is a link to one of them: https://pypi.org/project/pep8/

Reply:
right... It converts cells to .py file... but it doesn't check style, it is using the code in cells directly... I will check for those... thanks